site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,90 @@
+7. All configuration has to be possible using the UI of Mattermost.
+8. The plugin id defined in the manifest must not collide with the id of an existing plugin in the marketplace. It should follow [the naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
+
+**Technical Requirements (Checked by a developers of the Toolkit or the Integrations team)**
+
+1. The plugin works for 60k concurrent connections and in a high availability environment. (There are currently no tools available to verify this property. Hence, it is checked via code review by a developer)
+2. The plugin logs important events on appropriate log levels to allow system admins to troubleshoot issues.
+
+**Security Requirements (Checked by a member of the security team)**
+
+1. The plugin does not expose a vulnerability.
+2. The plugin does not include favor the author of the plugin or a third party excessively by e.g. including a bitcoin miner that mines on behalf of the author.
+3. The plugins author must notify Mattermost about any vulnerabilities in the future.
+
+**Functional Requirements (Checked by a and be avaiable  QA tester)**
+
+1. The plugin works as expected with the latest version of Mattermost.
+2. The plugin works as expected with the latest ESR version of Mattermost. This must not be checked if `min_server_version` is higher than the latest ESR version.
+
+Given that plugins are currently in Beta and rapid development is still happening, breaking changes especially for the webapp can happen. It is expected from plugin authors to keep up with these changes and fix bugs that may occur. Breaking changes will be mentioned in the "Important Upgrades Notes" section of the [Mattermost Changelog](https://docs.mattermost.com/administration/changelog.html).
+
+### Requirements for updating a community plugin to the Marketplace
+On every update of a community plugin must still fulfill the following checklist to be added to the Marketplace. This is checked by the four reviewers in the same way as when the plugin was added. The code review and security review should be performed against the `diff` of the last version in the Marketplace and the new version.
+
+The release also has to follow [semver](https://semver.org/). This specifically means for plugins:
+
+* If the plugin exposes an API for inter plugin communication, breaking changes to the API require a major version bump.
+* If an update requires manual migration actions from the administrator, a major version bump is required.
+
+This is checked in dev review.
+
+The new release must not change the plugin id defined in the manifest as this would require a reconfiguration of the plugin by a system admin.
+
+### Process for adding community plugin to the Marketplace
+There will be an owner for community plugins, that will guide you through the review process. Currently this is [hanzei](https://github.com/hanzei).
```

Comment:
```suggestion
All community plugins are assigned an owner to guide you through the review process. Connect with [hanzei](https://github.com/hanzei) for more details.
```

Also, should we link to your community handle instead?

Reply:
Given that the process takes place on GitHub I'm fine with having my GitHub username here.